codesm/formatter.py

```
@@ -159,6 +160,7 @@ async def _check_formatter_available(self, formatter: FormatterType) -> bool:
             self._available_formatters[formatter] = available
             return available
         except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)